libraries/c_sdk/standard/mqtt/src/private/iot_mqtt_internal.h

```diff
@@ -1076,4 +1103,117 @@ IotMqttError_t _IotMqtt_managedUnsubscribe( IotMqttConnection_t mqttConnection,
  */
 IotMqttError_t convertReturnCode( MQTTStatus_t managedMqttStatus );
 
+
+/*-----------------------------------Subscription and Operation array container functions----------------------*/
+
+/**
+ * @brief Insert the subscription in the subscription array.
```

Comment:
Please update the documentation to match the funationality. The function is getting the free index

Reply:
Addressed